src/components/Banner/Banner.less

```diff
@@ -0,0 +1,90 @@
+@banner: lucid-Banner;
```

Comment:
Either we should make these "name" variables more specific or we should do away with them.

I'm thinking either something more like `Banner-name` if we need to use a variable.